Sources/SwiftCrossUI/Environment/EnvironmentValues.swift

```diff
@@ -38,6 +38,9 @@ public struct EnvironmentValues {
     /// The scale factor of the current window.
     public var windowScaleFactor: Double
 
+    /// The type of on-screen keyboard to show when a text field is focused.
```

Comment:
Update to mention that it still affects auto complete on some non-mobile platforms.